### ISSUE TITLE
The name is so lame

### ISSUE THREAD

**Issue by littleboy0k** [vladimiry/ElectronMail#48]
The name email-securely-app doesn't sound like an attractive one. Would suggest changing that to something better.

**Comment by vladimiry** [comment]
The name of the project to some extent reflects its purpose. The project got already renamed once with v1 release and I don't think that time has come to rename it again. But I agree name could sound fancier. So I'm ok to keep the issue open for a while and maybe one day project will get a new name.

**Comment by vinyll** [comment]
I was about to build something similar and had started the organisation [ElectronMail](https://github.com/ElectronMail). If you're ever interested I could set you as maintainer.

I guess SecureMail or other name could be nice as well.

**Comment by trymeouteh** [comment]
ElectronMail has a nice ring to it.

**Comment by vladimiry** [comment]
ElectronMail sounds good to me too, thinking of renaming the project.

**Comment by vinyll** [comment]
You've been invited as owner of the [ElectronMail Organisation](https://github.com/ElectronMail) :)

**Comment by vladimiry** [comment]
@vinyll thanks for the invite. I'm not going to [transfer](https://help.github.com/en/articles/transferring-a-repository) the project to the [ElectronMail Organisation](https://github.com/ElectronMail) yet since I'm not sure what would be an implication of such move. I'm not really experienced of how GitHub works in terms of shared control. So I'm not accepting the invitation at this time. Would prefer to release the project with new name first and then we could figure other things out.

**Comment by vladimiry** [comment]
@vinyll also is that the organisation going to be used only for protonmail related open-source stuff? The app also supports tutanota service. Besides was you considering invite there projects like https://github.com/scastiel/protonmail-export and https://github.com/emersion/hydroxide?

**Comment by vladimiry** [comment]
The project has been renamed to ElectronMail. I'm going to polish some stuff and then publish v3.0.0 release, probably tomorrow.

**Comment by vinyll** [comment]
Feel free to use (or not) the organisation and publish there whichever repos make sense to you like protonmail export and hydroxide.
I initially had started that to build what you have very well done :)

Even though ElectronMail feels more linked to protonmail than Tutanota, it's still a mail app build with Proton…

Anyways, I'm glad to see the Electron mail popping up soon in my installed apps 👍

**Comment by vladimiry** [comment]
ElectronMail v3.0.0 release will try resolving `email-securely-app` settings and then if succeeded will propose importing the settings showing similar screen:

![migrating2](https://user-images.githubusercontent.com/1560781/54263030-d9940e00-4580-11e9-96a6-8589145752e8.png)

For Snap package copying the settings won't be possible due to the isolated environment, so the screen will look a little different:

![migrating-snap](https://user-images.githubusercontent.com/1560781/54262642-be74ce80-457f-11e9-9cdb-f8283c61d379.png)

**Comment by vladimiry** [comment]
ElectronMail v3.0.0 has just been released.

**Comment by vinyll** [comment]
@vladimiry if you want to keep the project location here, would you have any interest for [the logo](https://github.com/organizations/ElectronMail/settings/profile)?
I which case I can fine tune it and pass over the image and find back the svg.

**Comment by joshirio** [comment]
Hoping snap store will accept a rename or else I have to upload a new package with new name, not ideal for end users, request progress, not much hope for it since many renaming request were declined: https://forum.snapcraft.io/t/snap-rename-on-upstream-project-changes/10403

Regarding the AUR package, I'll repeat the same procedure done with the first rename (protonmail-desktop -> email-securely-app), which was to create a new AUR package and then request a package merge to a trusted user (TU).

**Comment by vladimiry** [comment]
> would you have any interest for the logo

I'd stick to the current icon for now since I don't think that it looks very ugly even though [being built with Libreoffice](https://github.com/vladimiry/ElectronMail/blob/2606ffb04f0d1512c2ba24c8393faf08ae070907/src/assets/icon/icon.odt) and then just printscreened :laughing: Besides changing the icon to something very different would probably involve changing the app's colors scheme which I'd prefer to avoid doing without actual need. But if you think there are better icon options I'd suggest opening a new issue. So after a while, maybe with v4 release, the app could get a new icon similar to how the name got changed.

@joshirio, it would be great if the Snap repository gets renamed but you are right it seems unlikely to happen. I will tweak the readme with new repositories links once renaming of both repositories is done. Thanks for keeping maintaining the repositories.

**Comment by vladimiry** [comment]
I think `Mailtron` name would also work, it's shorter and also correlates with electron/proton/mail terms. But no worries I'm not going to rename the project in the near future.

**Comment by Rudxain** [comment]
The obvious name should be **Hydrogen.** Why hasn't anyone suggested this?
